data/modules/Equipment/Internal.lua

```diff
@@ -0,0 +1,242 @@
+-- Copyright © 2008-2024 Pioneer Developers. See AUTHORS.txt for details
+-- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+local EquipTypes = require 'EquipType'
+local Equipment = require 'Equipment'
+
+local EquipType = EquipTypes.EquipType
+local SensorType = EquipTypes.SensorType
+local CabinType = EquipTypes.CabinType
+local ThrusterType = EquipTypes.ThrusterType
+
+--===============================================
+-- Computer Modules
+--===============================================
+
+Equipment.Register("misc.autopilot", EquipType.New {
+	l10n_key="AUTOPILOT",
+	price=1400, purchasable=true, tech_level=1,
+	slot = { type="computer.autopilot", size=1 },
+	mass=0.2, volume=0.5, capabilities = { set_speed=1, autopilot=1 },
+	icon_name="equip_autopilot"
+})
+
+Equipment.Register("misc.trade_computer", EquipType.New {
+	l10n_key="TRADE_COMPUTER",
+	price=400, purchasable=true, tech_level=9,
+	slot={ type="computer", size=1 },
```

Comment:
Should this be `type="computer.trade_computer"`, or, alternatively, should autopilot be `type="computer"`?

Reply:
Likely the former, however I'm deferring it to Stage 2 when we figure out / expand computer module gameplay and determine the restrictions on computer slots.